docs/environment-settings/environment.md

```
@@ -0,0 +1,78 @@
+Some Claudie aspects can be configured via environment variables read from a ConfigMap in the namespace where Claudie is deployed.
+
+These values let you to customize service endpoints, log levels, and concurrency limits. The default ConfigMap that is
+deployed with Claudie can be found in the [Claudie GitHub repository](https://github.com/berops/claudie/blob/master/manifests/claudie/.env).
+
+The full list of environment variables is as follows:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)